plugins/chat/assets/javascripts/discourse/components/chat-message.gjs

```diff
@@ -484,6 +485,19 @@ export default class ChatMessage extends Component {
     return this.args.context === MESSAGE_CONTEXT_THREAD;
   }
 
+  get shouldRenderStopMessageStreamingButton() {
+    return (
+      this.args.message.streaming &&
+      (this.currentUser.admin ||
+        this.args.message.user.id === this.currentUser.id)
```

Comment:
Does this need to account for `in_reply_to`?

Reply:
I think that potentially we could stream a message as a reply to another message if that's what you were asking?